### PR TITLE
rework of the control script

### PR DESCRIPTION
**Pull request by pierfabre**

### CODE REVIEW

**Comment by pierfabre** [comment]
ready for first round of review  @Armandpl 
Also what do you think of moving robot.py in robot/software?

File: scripts/control.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
+import json
+
+import matplotlib
+
+# Tkinter needs to be installed for matplotlib to work
+# sudo apt-get install python3-tk
+matplotlib.use("TkAgg")
```
Comment (Armandpl) [inline scripts/control.py:7]: maybe investigate why we need that?

File: scripts/control.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+    # Init motor controller
+    motor_controller = Controller.build_controller(parameters["motor_controller"])
+
+    # Reset encoders
+    robot.reset_encoders()
+    print("GO")
+
+    # Wait for user input to start the control loop
+    input()
```
Comment (Armandpl) [inline scripts/control.py:85]: add a message so that the user knows he should press a key

File: robot/software/controllers.py
Comment (Armandpl) [inline robot/software/controllers.py:1]: I think this belong to the furuta_gym package which we should rename furuta. This way we can ship all of the code to pypi.
I think the robot directory should contain only what's needed for the physical robot. library and control code should be in the python package.

**Review by Armandpl — CHANGES_REQUESTED** [review]
- need to move the library code to the furuta package
- later we might also want to move from manual instantiation from json files to using hydra but for now that's 100% ok
apart from that lgtm <3

File: robot/software/controllers.py
Comment (Armandpl) [reply robot/software/controllers.py:1]: i think the pattern for a user of the robot should be to create their own project, pip install furuta then setup their own experiments.

File: scripts/control.py (hunk 2 above)
Comment (Armandpl) [reply scripts/control.py:85]: when this was in a notebook it was obvious bc a modal would pop up, that's not longer the case in a script

File: scripts/control.py (hunk 1 above)
Comment (pierfabre) [reply scripts/control.py:7]: Will do during the test on the real robot

File: robot/software/controllers.py
Comment (pierfabre) [reply robot/software/controllers.py:1]: Discussed IRL I'm ok with this

File: scripts/control.py (hunk 1 above)
Comment (pierfabre) [reply scripts/control.py:7]: Tested, work without, don't know what happened last time, rm this part

**Review by Armandpl — APPROVED** [review]

**Pull request merged**